content/foundations/css-utilities/borders.mdx

```diff
@@ -30,7 +30,7 @@ You can adjust border widths on all sides or each side individually with respons
 - `border-(sm|md|lg|xl)` adds borders on all sides at and above the breakpoint. The `border-(sm|md|lg|xl)` shorthand is also supported.
 - `border-(sm|md|lg|xl)-0` removes borders from all sides at and above the breakpoint.
 - `border-(sm|md|lg|xl)-(top|right|bottom|left)` adds a border on the given side at and above the breakpoint.
-- `border-(sm|md|lg|xl)-(top|right|bottom|left)-0` the border from the given side at and above the breakpoint.
+- `border-(sm|md|lg|xl)-(top|right|bottom|left)-0` removes the border from the given side at and above the breakpoint.
```

Comment:
Sorry for being picky, but doesn't this set it to `0`? I am wondering if this may be an important distinction in some cases, e.g. an element has a default border in the browser. @francinelucca

Reply:
oh yeah, that makes sense, I'll update, thanks!